railties/lib/rails/commands/test.rb

```diff
@@ -2,4 +2,4 @@
 
 $: << File.expand_path("../../test", APP_PATH)
 
-Rails::TestRunner.run(ARGV)
+Minitest.autorun
```

Comment:
regarding the naming discussion `TestRequirer` and `TestRunner`. I'd rather opt for something, which does not rely on `Minitest.autorun`.

@kaspth do you see any reasons not to go down the path of explicitly calling `Minitest.run` when we want it to run? Also clearing in between runs, making it possible to run different sets of tests in the same process?